plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/admin/KickAdminCommand.java

```diff
@@ -49,7 +50,13 @@ public void execute(CommandSender sender, String[] args) {
         broadcastMessage(formatter.format(messages.broadcastKick));
         user.sendMessage(formatter.format(messages.kickToPlayer));
 
-        user.getPlayer().peek(player -> this.concurrencyManager.postRequests(new PrefixGlobalUpdatePlayer(player)));
+        Player player = server.getPlayer(user.getUUID());
+
+        if (player == null) {
+            return;
+        }
+
+        this.concurrencyManager.postRequests(new PrefixGlobalUpdatePlayer(player));
```

Comment:
Może żeby nie cofać się w rozwoju, to przykryć Server jakimś wrapperem, który zwraca Option<Player>, tak jak wcześniej robił to User.